examples/tptp/CMakeLists.txt

```diff
@@ -13,6 +13,22 @@ find_package(Z3
   # use this option.
   NO_DEFAULT_PATH
 )
+
+################################################################################
```

Comment:
they didn't before. I don't know if requiring C++11 here is an advantage: what if clients forget to require C++11. They their builds fail.

Reply:
We've been building the other examples with C++11 (I copied that from the `examples` `CMakeLists.txt`) ... also, in the 4.6 release notes, it indicates that C++11 is required for both building Z3 and using the C++ API.

Reply:
We could put a check for the C++ version in `z3++.h` and issue a warning / error if it is less than C++11?